packages/ui/src/working-groups/hooks/useOpenings.ts

```diff
@@ -1,17 +1,34 @@
+import { useMemo } from 'react'
+
 import { useGetWorkingGroupOpeningsQuery } from '../queries'
-import { asWorkingGroupOpening } from '../types'
+import { asWorkingGroupOpening, WorkingGroupOpening } from '../types'
 
 interface UseOpeningsParams {
   groupId?: string
+  type?: 'open' | 'past'
```

Comment:
I don't recall a part where we would need both, past and open, together. I'd go with `type: 'open' | 'past'`.